test/Microsoft.Extensions.Logging.Test/TestLogger.cs

```diff
@@ -31,7 +31,7 @@ public IDisposable BeginScopeImpl(object state)
                 Scope = state,
             });
 
-            return null;
+            return new NoopDisposable();
```

Comment:
This was to make sure that the test logger's scope was created when the func was executed